### PR TITLE
Feat fix

### PR DESCRIPTION
**Pull request by Kothalkarpriya**
Added footer, fixed all the links and footer styling

### CODE REVIEW

**Review by Avin008 — APPROVED** [review]
looks good to merge

File: assets/css/style.css [hunk 1]
```diff
@@ -116,7 +119,7 @@ i:hover {
 
 .main-section .container-horizon {
     margin: 1rem;
-    width: 40%;
+    max-width: 350px;
```
Comment (hussiantabha) [inline assets/css/style.css]: use rems

**Comment by hussiantabha** [comment]
Suggest a change and after changing it looks good to merge

File: assets/css/style.css (hunk 1 above)
Comment (Kothalkarpriya) [reply assets/css/style.css]: I will do it

**Review by neetukumari4858 — COMMENTED** [review]
it's good to merge

**Pull request merged**